activerecord/test/cases/associations/eager_test.rb

```diff
@@ -743,6 +743,18 @@ def test_eager_with_invalid_association_reference
     }
   end
 
+  def test_eager_has_many_through_with_order
+    tag = OrderedTag.create(:name => 'Foo')
+    post1 = Post.create!(:title => 'Beaches', :body => "I like beaches!")
+    post2 = Post.create!(:title => 'Pools', :body => "I like pools!")
+
+    Tagging.create!(:taggable_type => 'Post', :taggable_id => post1.id, :tag => tag)
+    Tagging.create!(:taggable_type => 'Post', :taggable_id => post2.id, :tag => tag)
```

Comment:
New tests should use the Ruby 1.9 hash syntax not hash rockets. So instead these should be changed to be like this: 

`Tagging.create!(taggable_type: 'Post', taggable_id: post2.id, tag: tag)`